domains/almaz.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "AlmaziikDev",
+        "email": "almazgamer228@gmail.com"
+    },
+    "record": {
+        "CNAME": "almaziikdev.github.io"
```

Comment:
Inaccessible website.

Reply:
Website is still inaccessible. Conversation is unresolved until the site is reachable.